internal/services/network/virtual_network_gateway_connection_data_source.go

```diff
@@ -119,6 +119,11 @@ func dataSourceVirtualNetworkGatewayConnection() *pluginsdk.Resource {
 				Computed: true,
 			},
 
+			"private_link_fast_path_enabled": {
+				Type:     pluginsdk.TypeBool,
+				Computed: true,
+			},
```

Comment:
Just realised this got added to the data source too. Could you please update the data source docs as well?

Reply:
Yes, handled in https://github.com/hashicorp/terraform-provider-azurerm/pull/25650/commits/59a0db9001438cac8b49e17e7903628d9f7c5bec